1.7/administration/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -0,0 +1,53 @@
+---
+post_title: Install Docker on CentOS
+menu_order: 2
+---
+Docker's <a href="https://docs.docker.com/engine/installation/linux/centos/" target="_blank">CentOS-specific installation instructions</a> are always going to be the most up to date for the latest version of Docker. However, because of the combination of using systemd and OverlayFS, it may be simpler to follow the following condensed steps.
+
```

Comment:
Can you suggest some specific language? I'm not sure how to express it. The reason is that there's a million ways to configure docker and the simplified instructions show how to configure systemd (which is required by DC/OS, thus a logical thing to monitor Docker too) and OverlayFS (which is recommended but not required).